bin/reth/src/config.rs

```diff
@@ -35,7 +35,7 @@ impl Config {
             .with_trusted_nodes(self.peers.trusted_nodes.clone())
             .with_connect_trusted_nodes_only(self.peers.connect_trusted_nodes_only);
         NetworkConfig::builder(Arc::new(ProviderImpl::new(db)), rng_secret_key())
-            .boot_nodes(bootnodes.unwrap_or_else(|| mainnet_nodes()))
+            .boot_nodes(bootnodes.clone().unwrap_or_else(mainnet_nodes))
```

Comment:
with that change the argument should not be a reference

Reply:
@mattsse Done